packages/interbit-ui-components/src/components/Navigation/HeaderGrid.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Nav } from 'react-bootstrap'
+
+import NavLinkWrapper from './NavLinkWrapper'
+import LinkWrapper from '../UIKit/LinkWrapper'
+import Logo from '../UIKit/Logo'
+
+export default class HeaderGrid extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    logo: PropTypes.shape({
+      logoEl: PropTypes.element,
+      to: PropTypes.string,
+      isDisabled: PropTypes.bool
+    }),
+    leftNavItems: PropTypes.arrayOf(
+      PropTypes.shape({
+        eventKey: PropTypes.string,
+        to: PropTypes.string,
+        text: PropTypes.string,
+        isHidden: PropTypes.bool
+      })
+    ),
+    rightNavItems: PropTypes.arrayOf(
+      PropTypes.shape({
+        eventKey: PropTypes.string,
+        to: PropTypes.string,
+        text: PropTypes.string,
+        isHidden: PropTypes.bool,
+        className: PropTypes.string,
+        id: PropTypes.string
+      })
+    ),
+    location: PropTypes.shape({})
+  }
+
+  static defaultProps = {
+    className: '',
+    logo: {
+      logoEl: <Logo className="sm hidden-xs" />,
+      to: '',
+      isDisabled: false
+    },
+    leftNavItems: [],
+    rightNavItems: [],
+    location: {}
+  }
+
+  render() {
+    const {
+      leftNavItems,
+      rightNavItems,
+      logo,
+      className,
+      location
+    } = this.props
+
+    return (
+      <div className={`ibweb-header ${className}`}>
+        <div className="grid-cont">
+          <div className="logo">
+            {logo.isDisabled ? (
+              /* TODO: adjust wordmark visibility based on new grid widths */
```

Comment:
Please make a ticket if there isn't one and add the ticket number to this comment here.

Reply:
Done and done 👍